app/src/main/kotlin/de/janhopp/luebeckmensawidget/ui/DateTopBar.kt

```diff
@@ -0,0 +1,33 @@
+package de.janhopp.luebeckmensawidget.ui
+
+import androidx.compose.material3.LocalTextStyle
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.glance.GlanceModifier
+import androidx.glance.layout.Alignment
+import androidx.glance.layout.Box
+import androidx.glance.layout.fillMaxWidth
+import androidx.glance.layout.padding
+import androidx.glance.text.FontWeight
+import de.janhopp.luebeckmensawidget.api.model.MensaDay
+import de.janhopp.luebeckmensawidget.theme.toGlance
+import de.janhopp.luebeckmensawidget.ui.components.StyledText
+import de.janhopp.luebeckmensawidget.utils.toDisplayString
+
+@Composable
+fun DateTopBar(day: MensaDay) {
+    Box(
+        modifier = GlanceModifier
+            .fillMaxWidth()
+            .padding(all = 8.dp)
+            .padding(top = 6.dp),
+        contentAlignment = Alignment.Center,
+    ) {
+        StyledText(
+            text = day.toDisplayString(),
+            style = LocalTextStyle.current.toGlance()
+                .copy(fontWeight = FontWeight.Bold, fontSize = 18.sp)
+        )
+    }
+}
```

Comment:
I think the date is a bit big and doesn't really fit with the rest of the layout. But this could maybe be a new issue.

Reply:
#48 opened for discussion and fix